lib/models/contact_page.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:hive/hive.dart';
+import 'package:hive_flutter/hive_flutter.dart';
+
+import 'contact.dart';
+
+class ContactPage extends StatelessWidget {
```

Comment:
What's this page used for? When I loaded it up on my device it was blank.
This also doesn't extend the AbstractPage class that handles layout and everything.